snapcraft/tests/commands/test_init.py

```diff
@@ -84,3 +84,44 @@ def test_init_with_existing_yaml(self):
 
         self.assertThat(str(raised), Equals(
             '{} already exists!'.format(self.yaml_path)))
+
+
+class InitCommandGitTestCase(CommandBaseTestCase):
+
+    def test_init_must_not_write_gitignore_if_no_git_dir(self):
+        self.run_command(['init'])
+
+        # Verify the .gitignore was created
```

Comment:
There's a typo here: Verify the .gitignore was NOT created.
I would just remove this comment, the assertion is clear enough, I think.